requirements/base.in

```diff
@@ -28,6 +28,10 @@ drf-writable-nested
 humanize
 drc-cmis
 
-commonground-api-common[setup-configuration, testutils, oas]
+#commonground-api-common[setup-configuration, testutils, oas]
+
+git+https://github.com/maykinmedia/commonground-api-common.git@main[setup-configuration, testutils, oas]
+git+https://github.com/maykinmedia/notifications-api-common.git@main
```

Comment:
Released commonground-api-common 2.8.0 and prepared a release for notifs-api-common